UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
+- Use descriptive text for button labels and links, and avoid labels such as "Learn more".
+- Use unique labels for distrinct actions.
+- Use existing components over creating new components unless you identify a need that is not addressed by the current components.
```

Comment:
Personally I'd go a bit farther here and suggest that even when a need _is_ identified that isn't addressed by an existing component (assuming we're talking about web components here), we should still consider extending the existing component first. Sometimes it won't make sense to do that (e.g., with components that are purpose-built, which is common and IMO fine), but in general we should lean toward extending those components that were built with reusability in mind.